hostfactory/host_provider/src/cluster.py

```diff
@@ -43,108 +41,49 @@ def status(self):
     
     def get_buckets(self):
         buckets = self.node_mgr.get_buckets()
-        status_json = self.status()
-        nodearrays = status_json["nodearrays"]
-        for nodearray_root in nodearrays:
-            for bucket in buckets:
-                if nodearray_root.get("name") == bucket.nodearray:
-                    bucket.priority = nodearray_root.get("Priority")
+        self.logger.debug("Buckets %s", buckets)        
```

Comment:
This is a good idea to log this, but I think we should include a bit more information - specifically the bucket.limits object, which gives a detailed breakdown of how we come up with the available_count. I am focusing on this because I think this is exactly when we would expect a user to be dealing with capacity issues, as well we can ask a user to simply run this and check the logs.

        self.logger.debug("Buckets: count=%d", len(buckets))
        for b in buckets:
            self.logger.debug(f"{b.nodearray}/{b.vm_size} available={b.available_count}" +
                              " based on {b.limits} bucket_id={b.bucket_id}")